client/my-sites/site-settings/form-general.jsx

```diff
@@ -698,6 +707,36 @@ export class SiteSettingsFormGeneral extends Component {
 			</div>
 		);
 	}
+
+	advancedCustomizationNotice() {
+		const { translate, selectedSite, siteSlug } = this.props;
+		const upgradeUrl = `/plans/${ siteSlug }`;
+
+		return (
+			<>
+				<div className="site-settings__advanced-customization-notice">
+					<div className="site-settings__advanced-customization-notice-cta">
+						<Gridicon icon="info-outline" />
+						<span>
+							{ translate( "Your style changes won't be public until you upgrade your plan." ) }
+						</span>
+					</div>
+					<div className="site-settings__advanced-customization-notice-buttons">
+						<Button href={ selectedSite.URL } target="_blank">
+							{ translate( 'View site' ) }
+						</Button>
+						<Button
+							className="is-primary"
```

Comment:
Nitpick (non-blocking): The same can be accomplished by setting a `primary` prop.
```suggestion
							primary
```